guides/common/modules/ref_predefined-roles.adoc

```diff
@@ -27,9 +27,9 @@ In the {ProjectWebUI}, global settings can be found under *Administer* > *Settin
 | Organization admin | An administrator role defined per organization.
 The role has no visibility into resources in other organizations.
 | Red{nbsp}Hat Access Logs | View the log viewer and the logs.
-| Remote Execution Manager | A role with full remote execution permissions, including modifying job templates.
-| Remote Execution User | Run remote execution jobs.
-| Site manager | A restrained version of the Manager role.
+| Remote Execution Manager | Full remote execution permissions, excluding {ProjectServers} and {SmartProxies}
+| Remote Execution User | Run remote execution jobs against hosts, excluding {ProjectServers} and {SmartProxies}.
```

Comment:
This is the sentence that breaks parallelism with the rest of your entries here that I flagged in my previous review.